docs/guides/deploy-hyperlane-local-agents.mdx

```diff
@@ -6,6 +6,10 @@ import SendTestMessagesPartial from "/docs/partials/deploy-hyperlane/_send-test-
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
+:::danger
+This page is out of date. Although the concepts are still relevant, some CLI commands may not be available.
+:::
```

Comment:
This feels like a mistake. Either lets update the page or cut it. Having this here makes it seem like we've abandoned maintenance our own docs